packages/components/dialog/src/dialog.js

```diff
@@ -22,20 +31,26 @@ export class TSDialog extends TSElement {
 
 	static get properties() {
 		return {
-			/** Dialog can be toggled by add/removing this attribute */
+			/** Dialog can be toggled by adding/removing this attribute */
 			visible: { type: Boolean, attribute: 'data-visible', reflect: true },
 			/** Text content of the modal */
 			text: { type: String, reflect: true },
-			/** If you need a different icon that default ones, you can use one of Elements icon names */
+			/** If you need a different icon that default ones, you can use one of Elements icon names. Notifications will ignore this */
 			icon: { type: String, reflect: true },
-			/** `confirm`, `warning`, `danger` */
+			/** `success`, `info`, `confirm`, `warning`, `danger`, `error` */
 			type: { type: String, reflect: true },
 			/** can be used for customizing the buttons text and translations */
 			translations: { type: Object, reflect: true },
 			/** set the default focus on the button, either `accept` or `cancel` */
 			focused: { type: String, reflect: true },
 			/** either `accept` or `cancel` can be used to change the button type, based on the dialog type, by default both are secondary */
 			primary: { type: String, reflect: true },
+			/** If it is a notification, no cancel button will be rendered. Notifications of type 'success' will auto-close on timeout, if they are not `non-dismissable` */
+			notification: { type: Boolean, reflect: true },
+			/** Render no buttons. This only affects notifications of type 'success' */
+			noButtons: { type: Boolean, reflect: true, attribute: 'no-buttons' },
```

Comment:
I think that is an excellent suggestion! The noButtons is a notifications-only, even only for success type, prop, and you should always try to avoid that. Just easily happens when when merging two components into one.